### ISSUE TITLE
Remove inventory

### ISSUE THREAD

**Issue by Gireen** [Unvanquished/Unvanquished#1704]
It is not documented anywhere so i write down this issue.
The inventory was or is planed to be removed. This is a process which date back to the beginning of unv and is now finally doable.
Through the auto use jetpack and default binds for weapons,grenade and medikit. It is now possible to use all items and swap weapons without the use of a mousewheel.

The remaining icons can be moved to left and right bottom part of the hud.
Armor and medikit can be right of the health bar.
Jetpack at the place where its displaced on activation.
Grenades and radar can be on the left side of the ammo clips.

**Comment by bmorel** [comment]
Just to be certain: by "inventory", you mean the list of hard to see, small, white icons on the left part of human HUD, this:
![2022-01-13-213640_1362x748_scrot](https://user-images.githubusercontent.com/1316300/149405821-f5ce0cde-4f4e-46df-8951-df74b8f7e963.png)

If yes, I'm totally in favor!
If not, please explain more :)

**Comment by Gireen** [comment]
Yes. I think it also still has its old function, that if you scroll and middle click you switch or activate the selected item.
